lib/core-generators/new/templates/config/custom.js.template

```diff
@@ -90,7 +90,16 @@ module.exports.custom = {
   *                                                                          *
   * Any other custom config this Sails app should use during development.    *
   *                                                                          *
-  ***************************************************************************/<% if (!caviar) { %>
+  ***************************************************************************/
+  
+  platformCopyrightYear: '2019',
+  platformName:  'NEW_APP_NAME',
+  platformCompanyName: 'NEW_APP_COMPANY_NAME',
+  platformCompanyAboutHref: 'https://sailsjs.com/about',
+  privacyPolicyUpdatedAt: 'July 16, 2019',
+  termsOfServiceUpdatedAt: 'July 16, 2019'
+
+  <% if (!caviar) { %>
```

Comment:
The new custom config items should go inside this `if`, since they're only used in the Web App template